qemu/tests/single_driver_install.py

```diff
@@ -127,9 +127,13 @@ def run(test, params, env):
                                           pnp_cmd) % inf_name
             status, output = session.cmd_status_output(uninst_store_cmd,
                                                        inst_timeout)
-            if status:
+            if status not in (0, 3010):
                 test.error("Failed to uninstall driver '%s' from store, "
                            "details:\n%s" % (driver_name, output))
+            if params.get_boolean('need_reboot'):
+                # for viostor and vioscsi, they need system reboot
+                # acceptable status: OK(0), REBOOT(3010)
+                session = vm.reboot(session)
```

Comment:
@menli820 Hi, just curious that if status is OK, do we still need system reboot?

Reply:
@xiagao for some devices eg rng ,vioser  we can use this "pnputil /delete-driver %s /uninstall /force" to finish the uninstall Driver package successfuly,  so status == 0, and we don't need to system reboot.


For storage, the output like the following, so we need to reboot the system.
[stdout] [stdout] Driver package uninstalled. 
[stdout] Driver package deleted successfully. 
[stdout] System reboot is needed to complete unconfiguration operations! [stdout]

Reply:
I saw there are steps to reboot or destroy/boot up vm in the following lines. So that how about delete L133-L136.

Reply:
@xiagao Thanks for the suggestion, it is more reasonable, update the code.

 (1/1) Host_RHEL.m9.u5.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.win_virtio_driver_update_test.with_viostor.during_bg_test.uninstall_install.q35: STARTED
 (1/1) Host_RHEL.m9.u5.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.win_virtio_driver_update_test.with_viostor.during_bg_test.uninstall_install.q35: PASS (364.85 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0